README.md

```diff
@@ -100,17 +100,13 @@ to pass into your GitHub Workflow.
 
 See [action.yml](action.yml) for the latest interface and docs.
 
-### Run a self-contained notebook
-The workflow below runs a self-contained notebook as a one-time job.
+### Run notebook within a temporary checkout of the current Repo
```

Comment:
```suggestion
### (Recommended) Run notebook within a temporary checkout of the current Repo
```

Mark this option as recommended? Since with the Jobs repos integration this is now our recommended approach